packages/jaeger-ui/src/utils/readJsonFile.tsx

```diff
@@ -14,6 +14,22 @@
 
 import JaegerAPI from '../api/jaeger';
 
+function tryParseMultiLineInput(input: string): any[] {
+  const jsonStrings = input.split('\n').filter((line: string) => line.trim() !== '');
+  const parsedObjects: any[] = [];
+
+  jsonStrings.forEach((jsonString: string, index: number) => {
+    try {
+      const traceObj = JSON.parse(jsonString.trim());
+      parsedObjects.push(traceObj);
```

Comment:
How is it possible for this line not to have code coverage given the `'loads JSON-per-line data'` unit test? It makes me doubt that the test is valid, which is why I wanted to do more validation on the result.

Reply:
I was wondering the same thing. I'll debug it later tonight.